bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/digitalinvoice/onboarding/DigitalInvoiceOnboardingFragment.kt

```diff
@@ -130,17 +131,26 @@ class DigitalInvoiceOnboardingFragment : Fragment(), DigitalOnboardingScreenCont
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         setInputHandlers()
+        setupImageIllustrationAdapter()
         setupOnboardingBottomNavigationBar()
     }
 
 
+    private fun setupImageIllustrationAdapter() {
+        if (GiniCapture.hasInstance()) {
+            binding.digitalInvoiceImageContainer.injectedViewAdapter =
+                GiniBank.digitalInvoiceOnboardingIllustrationAdapter
```

Comment:
[Request] We also need to call `onVisible` when the fragment starts and `onHidden` when the fragment stops.